frontend/appflowy_flutter/packages/flowy_infra/lib/plugins/bloc/dynamic_plugin_bloc.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:bloc/bloc.dart';
+import 'package:flowy_infra/plugins/service/plugin_service.dart';
+
+import '../../file_picker/file_picker_impl.dart';
+import 'dynamic_plugin_event.dart';
+import 'dynamic_plugin_state.dart';
+
+class DynamicPluginBloc extends Bloc<DynamicPluginEvent, DynamicPluginState> {
```

Comment:
I prefer placing the event and state in the Bloc file. However, it's also acceptable to separate them.